onnxruntime/contrib_ops/rocm/bert/gemm_fast_gelu_tunable_op.h

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <hip/hip_runtime.h>
+#include <memory>
+#include <string>
+
+#include "contrib_ops/rocm/bert/fast_gelu_impl.h"
+#include "core/providers/rocm/tunable/gemm.h"
+#include "core/providers/rocm/tunable/gemm_common.h"
+#include "core/providers/rocm/tunable/rocm_tunable.h"
+
+using onnxruntime::rocm::tunable::blas::BlasOp;
+using onnxruntime::rocm::tunable::blas::BlasOpToString;
+using onnxruntime::rocm::tunable::blas::column_major::Gemm;
+
+namespace onnxruntime {
+namespace contrib {
+namespace rocm {
+
+template <typename T>
+struct GemmFastGeluParams : onnxruntime::rocm::tunable::OpParams {
+  std::string Signature() const override {
+    return MakeString(BlasOpToString(opa), BlasOpToString(opb), "_", m, "_", n, "_", k);
+  }
+  rocblas_handle handle;
+  BlasOp opa;
+  BlasOp opb;
+  int64_t m;
+  int64_t n;
+  int64_t k;
+  T alpha;
+  const T* a;
+  int64_t lda;
+  const T* b;
+  int64_t ldb;
+  const T* bias;
+  T beta;
+  T* c;
+  int64_t ldc;
+  bool has_bias{true};
+  bool tuning{false};
```

Comment:
Backlog: It is weird that we need to put `tuning` flag in the `GemmFastGeluParams`. The reason is tuning is not a controlled behavior of underlying kernels/ops. It is the behavior of a TunableOp instance. We need to come up with a way to eliminate the transitive `tuning` passing here. 

Maybe remove the `op.EnableTuning()` interface and opt to use `params->tuning`. We need more discuss about this in the future.